addons/account/models/account_journal_dashboard.py

```diff
@@ -518,7 +518,7 @@ def open_action_with_context(self):
         ctx = dict(self.env.context, default_journal_id=self.id)
         if ctx.get('search_default_journal', False):
             ctx.update(search_default_journal_id=self.id)
-            del ctx['search_default_journal']  # otherwise it will do a useless groupby in bank statements
+            ctx['search_default_journal'] = False  # otherwise it will do a useless groupby in bank statements
```

Comment:
In order to get rid of the useless groupby journal why don't we pass a False value to 'search_default_journal' while calling the action in the 'account_journal_dashboard_view.xml' file, like:

`<a t-if="journal_type == 'bank'" role="menuitem" type="object" name="open_action_with_context" context="{'action_name': 'action_bank_statement_tree', 'search_default_journal': False}">Statements</a>`

Reply:
because it is needed in `open_action_with_context`
```python
        if ctx.get('search_default_journal', False):
            ctx.update(search_default_journal_id=self.id)
```
We won't have the correct journal selected in the search view without it

Reply:
Thanks @william-andre for your quick response and contribution. Yes, now I noticed it.
I think it's working well right now.